### ISSUE TITLE
Update transitive dependency mkdirp 0.5.2

### ISSUE THREAD

**Issue by pastelmind** [bahmutov/snap-shot-it#514]
`snap-shot-it` depends on `snap-shot-core`, which depends on `mkdirp`. Version 0.5.2 of `mkdirp` used to depend on `minimist` 0.0.8, which was recently marked as vulnerable by CVE-2020-7598. `mkdirp` 0.5.3 has recently been released to address the issue while maintaining backwards compatibility (`mkdirp` 1.x line has incompatible API).

Could you please update `mkdirp` to 0.5.3?

I also considered creating this issue in snap-shot-core's repo, but since I use snap-shot-it, I thought it would be better to post this here.

**Comment by Jolg42** [comment]
Indeed, @bahmutov could you merge this? https://github.com/bahmutov/snap-shot-core/pull/328 😃
